dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java

```diff
@@ -0,0 +1,160 @@
+package com.dotcms.contenttype.business;
+
+import com.dotcms.contenttype.business.FieldAPIImpl;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No need to import a type that lives in the same package](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)